adapters/repos/db/vector/hnsw/periodic_tombstone_removal_test.go

```diff
@@ -35,7 +35,7 @@ func TestPeriodicTombstoneRemoval(t *testing.T) {
 	require.Nil(t, err)
 
 	for i, vec := range testVectors {
-		err := index.Add(i, vec)
+		err := index.Add(int64(i), vec)
```

Comment:
whole file: see above, all node ids should be represented by `uint64`, removing the need for the conversions.